packages/syntax-core/src/Popover/Popover.tsx

```diff
@@ -0,0 +1,347 @@
+  type ComponentProps,
+} from "react";
+import { type Placement as RAPlacement } from "react-aria";
+import {
+  Popover as RACPopover,
+  DialogTrigger as RACDialogTrigger,
+  Dialog as RACDialog,
+  Modal as RACModal,
+  ModalOverlay as RACModalOverlay,
+} from "react-aria-components";
+import Box from "../Box/Box";
+import Typography from "../Typography/Typography";
+import IconButton from "../IconButton/IconButton";
+import styles from "./Popover.module.css";
+import Triggerable from "../react-aria-utils/Triggerable";
+import OverlayVisibility from "../react-aria-utils/OverlayVisibility";
+
+type Placement = "top" | "end" | "bottom" | "start";
+
+const SYNTAX_PLACEMENT_TO_RAC_PLACEMENT: Record<Placement, RAPlacement> = {
+  top: "top start",
+  end: "top end",
+  bottom: "bottom end",
+  start: "bottom start",
+} as const;
+
+function syntaxToRAPlacement(placement?: Placement): RAPlacement | undefined {
+  if (!placement) return undefined;
+  return SYNTAX_PLACEMENT_TO_RAC_PLACEMENT[placement];
+}
+
+/**
+ * [Popover](https://cambly-syntax.vercel.app/?path=/docs/components-popover--docs) displays contextual information on hover or focus.
+ *
+ * Popover content is hidden by default and shown on hover or focus.
+ * The content is hidden again when the user mouses out of the trigger element or blurs the trigger element or presses Escape
+ *
+ * Example Usage:
+ ```
+  <Popover
+    placement="bottom-start"
+    initialOpen
+    content={(
+      <Box padding={2} maxWidth={400}>
+        ... some content goes here
+      </Box>
+    )}
+  >
+      <Button text="Trigger me" />
+  </Popover>
+ ```
+ */
+const Popover = forwardRef<
+  HTMLDivElement,
+  {
+    /** Test id for the floating dialog */
+    "data-testid"?: string;
+    /** Optional aria-label for the popover (content element) */
+    accessibilityLabel?: string;
+    /** Optional trigger element */
+    children?: ReactElement | string;
+    /** Content to be shown inside the popover. */
+    content: ReactNode;
+    /**
+     * If set to true the popover will render initially open
+     * @defaultValue false
+     */
+    initialOpen?: boolean;
+    /** Optional boolean to control whether popover content is rendered as a modal */
+    modal?: boolean;
+    /** Optional handler for change of visibility for popover content, for analytics and control */
+    onChangeContentVisibility?: (visible: boolean) => void;
+    /** Optional boolean to control open state of popover externally */
+    open?: boolean;
+    /**
+     * Location of the popover content relative to anchor element
+     * @defaultValue "top-start"
+     */
+    placement?: Placement;
+  }
+>(function Popover(props, ref): ReactElement {
+  const {
+    "data-testid": dataTestId,
+    accessibilityLabel,
+    children,
+    content,
+    initialOpen,
+    modal: modalProp,
+    onChangeContentVisibility,
+    open,
+    placement = "bottom",
+  } = props;
+
+  const anchorNode =
+    typeof children === "string" ? (
```

Comment:
addressed in https://github.com/Cambly/syntax/pull/303/commits/c9777c0326e326cb9352dab43dcc3de80c6b1980

- Popover removes support for children as string
- Tooltip retains support for children as string, but wraps with raw span instead of `<Typography />` component.

I'm torn between removing support for children-as-string in Tooltip, and couldn't seem to bring myself to rip it out entirely.

@christianvuerings @somethiiing if either of you think that we should remove the Tooltip raw-string-as-trigger support, add a reply here saying so and I'll rip that out before merge as well!

Reply:
On second thought, scratch that.  It's easier to add this support back in in the future with a minor change release than it would be to remove that support through a major breaking change:  https://github.com/Cambly/syntax/pull/303/commits/b8ca635b943e0411a75f525a1023c34dc124e885